api-proxy.js

```diff
@@ -0,0 +1,49 @@
+var requestProxy = require('express-request-proxy');
```

Comment:
This is a great use of proxying your services, and routing your app functionality.